onnx/defs/reduction/defs.cc

```diff
@@ -21,20 +21,34 @@ std::vector<std::string> GetSupportedDataTypesForReductionOps(
   return OpSchema::numeric_types_for_math_reduction_with_bfloat();
 }
 
+inline std::string GenerateGeneralRedunctionDoc(std::string name) {
+  return "Computes the " + name + " of the input tensor's element along the provided axes. The resulted"
+"tensor has the same rank as the input if keepdims equal 1. If keepdims equal 0, then"
+"the resulted tensor have the reduced dimension pruned.\n"
+"The above behavior is similar to numpy, with the exception that numpy default keepdims to"
+"False instead of True.";
+}
+
+inline std::string GenerateComplementAxesDoc() {
+  return "\nThe above behavior is similar to numpy, with the following exceptions:"
```

Comment:
This line and the next is present in GenerateGeneralRedunctionDoc as well. I suggest keep it simple just keep 2 separate strings no need for concatenation